components/shared/profile/edit/EditProfile.tsx

```diff
@@ -0,0 +1,451 @@
+                {...register(`experiences[${index}].name` as any)}
+              />
+            </div>
+          </>
+        );
+      })}
+      <CustomButton
+        buttonType="profileButton"
+        type="button"
+        onClick={() => appendExperience({ name: "" })}
+      >
+        Add Experience
+      </CustomButton>
+      {/* Technologies Section with Search Box */}
+      <div className="flex flex-col justify-stretch gap-2 px-3.5 py-3  ">
+        <label
+          htmlFor="technologies"
+          className="justify-start space-y-2 text-white-300"
+        >
+          Tech Stacks
+        </label>
+        <div className="flex w-full  bg-black-700 ">
+          {technologies && // eslint-disable-next-line array-callback-return
+            technologies?.map((tech: any, index) => {
+              const icon = techStackBadges.find((badge) => badge.name === tech);
+              if (icon)
+                return (
+                  <>
+                    <span
+                      key={index}
+                      className="tech-badges paragraph-3-medium shadow-custom w-1/2   flex-row justify-between bg-black-600 text-white-100"
+                    >
+                      {icon.icon({ size: 16 })}
+                      {tech}
+                    </span>
+                  </>
+                );
+            })}
+
+          <Input
+            className=" w-1/2  bg-black-700 text-white-100"
+            value={search}
+            onChange={(e) => setSearch(e.target.value)}
+          />
+        </div>
+        <div className="flex h-9 w-full flex-row">
+          {search &&
+            results.length > 0 &&
+            // eslint-disable-next-line array-callback-return
+            results.map((result: any, index) => {
+              if (technologies?.includes(result.name)) {
+                return false;
+              }
+              const icon = techStackBadges.find(
+                (badge) => badge.name === result.name
+              );
+              if (icon)
+                return (
+                  <>
+                    <div className="flex  flex-col bg-black-700" key={index}>
+                      <Button
+                        key={index}
+                        className="text-white-100"
+                        onClick={(e) =>
+                          setValue(`technologies`, [
+                            ...technologies!,
+                            result.name,
+                          ])
+                        }
+                      >
+                        <div
+                          key={index}
+                          className="h-8 flex-row gap-x-[12px] rounded-[3px] bg-black-600 px-2  py-0.5"
+                        >
+                          <span className="tech-badges paragraph-3-medium grow">
+                            {icon.icon({ size: 16 })}
+                            {result.name}
+                          </span>
+                        </div>
```

Comment:
```suggestion
   <div
                        key={index}
                        className="flex h-8 flex-row items-center gap-x-[12px] rounded-[3px] bg-black-600 px-2  py-0.5"
                      >
                        {icon?.icon}
                        <span className="paragraph-3-medium">
                          {result.name}
                        </span>
                      </div>
```

I think this is closer to what you wanted. You can't have a flex-row without a flex. 

![image](https://github.com/agordon123/git_note/assets/4108484/da36dd7b-fcd4-4f0c-ac8f-7d34f1a1b6e1)

Reply:
thank you this was bothering me =)